packages/shortest/tests/e2e/index.ts

```diff
@@ -0,0 +1,96 @@
+import fs from "node:fs";
+import path from "node:path";
+import { describe, test, before } from "node:test";
+import { fileURLToPath } from "node:url";
+import { glob } from "glob";
+import pc from "picocolors";
+import { initializeConfig } from "@/index";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+function findMonorepoRoot(startDir: string): string {
+  let currentDir = startDir;
+  while (currentDir !== path.parse(currentDir).root) {
+    const pkgJsonPath = path.join(currentDir, "package.json");
+    if (fs.existsSync(pkgJsonPath)) {
+      const pkg = JSON.parse(fs.readFileSync(pkgJsonPath, "utf8"));
+      // Check if this is the root workspace package.json
+      if (pkg.workspaces || pkg.private) {
+        return currentDir;
+      }
+    }
+    currentDir = path.dirname(currentDir);
+  }
+  throw new Error(
+    "Could not find monorepo root (no workspace package.json found)",
+  );
+}
+
+const projectRoot = findMonorepoRoot(__dirname);
+
+/**
+ * E2E Test Runner
```

Comment:
Simple runner to run e2e tests for Shortest source code. Most specs are broken at this moment, will look into fixing them in a separate PR.